### PR TITLE
Refactoring tfsef for vulberabilities

### PR DESCRIPTION
**Pull request by jdesouza**
This PR fixes #

## Checklist
* [X] I have signed the CLA
* [X] I have updated/added any relevant documentation

### CODE REVIEW

File: plugins/ci/Dockerfile [hunk 1]
```diff
@@ -11,10 +11,15 @@ ENV helmVersion=3.11.2
...
 RUN if [ "${TARGETARCH}" = "amd64" ] ; then trivyArch="64bit"; else trivyArch="${TARGETARCH}"; fi && \
   curl -L https://github.com/aquasecurity/trivy/releases/download/v${trivyVersion}/trivy_${trivyVersion}_${TARGETOS}-${trivyArch}.tar.gz > trivy.tar.gz && tar -xvf trivy.tar.gz && mv ./trivy /usr/local/bin/trivy && rm trivy.tar.gz
-RUN curl -L https://github.com/aquasecurity/tfsec/releases/download/v${tfsecVersion}/tfsec-${TARGETOS}-${TARGETARCH} > /usr/local/bin/tfsec && chmod +x /usr/local/bin/tfsec
 RUN curl -L "https://github.com/FairwindsOps/polaris/releases/download/$polarisVersion/polaris_${TARGETOS}_${TARGETARCH}.tar.gz" > polaris.tar.gz && tar -xvf polaris.tar.gz && chmod +x polaris && rm polaris.tar.gz && mv ./polaris /usr/local/bin/polaris
 RUN curl -L "https://github.com/FairwindsOps/pluto/releases/download/v$plutoVersion/pluto_${plutoVersion}_${TARGETOS}_${TARGETARCH}.tar.gz" > pluto.tar.gz && tar -xvf pluto.tar.gz && chmod +x pluto && rm pluto.tar.gz && mv ./pluto /usr/local/bin/pluto
 
+RUN curl -L "https://github.com/aquasecurity/tfsec/archive/refs/tags/v${tfsecVersion}.tar.gz" > tfsec.tar.gz && tar -xvf tfsec.tar.gz
```
Comment (vitorvezani) [inline plugins/ci/Dockerfile:17]: We are not taking into account `${TARGETOS}` and `${TARGETARCH}` -- I suppose that will have some implication...

Comment (jdesouza) [reply plugins/ci/Dockerfile:17]: This is the source

Comment (jdesouza) [reply plugins/ci/Dockerfile:19]: It is to be compiled by go yet

**Comment by jdesouza** [comment]
This is not building :-\

**Pull request closed without merging**